ci-operator/step-registry/redhat-lp-chaos/vm/check/redhat-lp-chaos-vm-check-commands.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+set -euxo pipefail
```

Comment:
I just found out that the `erreixt`, i.e. `set -e` is not inherited by child in `bash` by default, unless `inherit_errexit` is set.

```suggestion
set -euxo pipefail; shopt -s inherit_errexit
```